instructor/common.py

```diff
@@ -680,6 +737,9 @@ def western_blot_band_size(request):
             update_wb_bands(assignment, antibodies=antibodies)
 
             if 'continue' in request.POST:
+                if assignment.last_enabled_page <= page_number:
+                    assignment.last_enabled_page = page_number+1
+                    assignment.save()
```

Comment:
You have this code block in a lot of places. You may want to refactor it into a helper of some kind.